src/scripts/run-td-extract.sh

```diff
@@ -64,6 +64,13 @@ case $key in
     shift # past argument
     shift # past value
     ;;
+    --)
+    shift
+    break 2
+    ;;
+    *)
+    echo "Unknown parameter $1."
```

Comment:
Just want to confirm, is the run-td-extract.sh intended for the most basic use only? Because we have many features whose flags will all just cause this script to fail.
If it is indeed intended so, I can make another change to update the README illustrating the option to use dwh-assessment-extraciton-tool.sh with full features.

Reply:
Yes it is. However, this change also add the option to pass through extra parameters after `--`.